setup.py

```diff
@@ -31,7 +31,7 @@
     ],
     extras_require={
         'uwsgi': ['uWSGI'],
-        'internal': ['yelp_meteorite']
+        'internal': ['yelp_meteorite', 'monk']
```

Comment:
Would it be possible to add a version number here? In my test this wasn't working because of a different Monk client lib version.

Reply:
I guess you mean a min version ie monk >= ?

Reply:
I think we should use either `==` or both `>` and `<=`, to ensure that when we release a new incompatible version of the monk client lib, clog will keep working.